client/src/app/collections/status-collection.coffee

```diff
@@ -0,0 +1,9 @@
+$ = require('jquery')
+Backbone = require('backbone')
+StatusModel = require('../models/status')
+
+module.exports = Backbone.Collection.extend(
+  model: StatusModel
+  comparator: (model) ->
+    -model.get('id')
```

Comment:
I was doing the sorting to have the most recent checks be at the top, which I figured was better for usability.  Without the sorting the most recent checks would be at the bottom.

If we can still have uniqueness on the back end without the md5 key, then we probably don't need it now.